haddock-api/resources/html/Linuwial.std-theme/linuwial.css

```diff
@@ -234,7 +234,7 @@ Display the package name on top of the menu links and center both elements:
  */
 
  body, button {
-   font: 400 15px/1.4 'PT Sans',
+   font: 400 14px/1.4 'PT Sans',
```

Comment:
font size on Hackage seems to be 16, 17px check the description of e.g. https://hackage.haskell.org/package/sop-core, so 14 imho too small in comparison. (12pt seems to be 16px, and 11pt 15px) e.g. https://everythingfonts.com/font/tools/units/pt-to-px - I wouldn't go under 11pt.

Reply:
Sure, it is for the project summary., but keep in mind that we are jumping from `font: 13px/1.4 sans-serif;` for the current actual docs.

Reply:
@harpocrates fair point. Would be nice to harmonise font-size at some point though.